streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -222,9 +222,9 @@ public void shouldDriveGlobalStore() throws Exception {
     }
 
     @Test
-    public void testDrivingSimpleMultiSourceTopology() {
+    public void testDrivingSimpleMultiSourceTopology() throws Exception {
```

Comment:
+1 to leave in.  IMHO it makes the test more readable without the try/catch boilerplate.